src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const payment = 40;
+  let sum = days * payment;
+
+  if (days > 6) {
```

Comment:
2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values

and for the next lines